theme/header.hbs

```diff
@@ -6,3 +6,10 @@
 <!-- To automatically render math in text elements, include the auto-render extension: -->
 <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/contrib/auto-render.min.js" integrity="sha384-yACMu8JWxKzSp/C1YV86pzGiQ/l1YUfE8oPuahJQxzehAjEt2GiQuy/BIvl9KyeF" crossorigin="anonymous"
     onload="renderMathInElement(document.body);"></script>
+
+<!-- Redirect to the overview page if on home, otherwise the image won't work on one of the following: local index, prod index, prod chapter -->
+<script defer>
```

Comment:
- looks like defer attribute is not needed
- it should be placed before other assets?
- what's the purpose of matching path `/deep-dive-tikv/`?

Reply:
We only want it to cover the URL https://tikv.github.io/deep-dive-tikv/, and we can't directly edit the index.html. Though from your comment I've realized I can localize this just to the particular section.. 😁 

I'll make a change, thanks!